graspologic/utils/utils.py

```diff
@@ -101,7 +101,7 @@ def import_edgelist(
         an array is returned.
 
     vertices : array-like, shape (n_vertices, )
-        If ``return_vertices`` == True, then returns an array of all vertices that were
+        If ``return_vertices == True``, then returns an array of all vertices that were
```

Comment:
is this really the convention? looks weirdish to wrap True insdide of ticks. maybe If ``return_vertices`` is set to True? what do we do everywhere else?

Reply:
I was interpreting the statement as code, but I agree that it looks weird, maybe " ``return_vertices`` is True "?